model/factories.go

```diff
@@ -125,3 +125,113 @@ func WaterTemperatureSensor() *SensorV2 {
 		Format:   keyRound1,
 	}.SensorV2()
 }
+
+// Variable factories //
+
+// NewAlarmNetworkVar returns a new AlarmNetwork var with the
+// passed number of failures as its value.
+func NewAlarmNetworkVar(numberOfFailures string) *Variable {
```

Comment:
Although Variable values are all represented as strings, it would be preferable if the constructors took the underlying type, e.g., an `int` in the case of `numberOfFailures`, and performed the conversion.